src/maintainer/knowledge_base.rst

```diff
@@ -1586,13 +1586,28 @@ Another reason why it is good to keep the PR open or in draft status is that peo
 You are also free to download recipes and rebuild them yourself, if you would like at least that much oversight. However, many people use ``conda-forge`` all the time with no issues and here are some things that ``conda-forge`` does to help with security in some ways:
 
 1. `Sources <https://conda-forge.org/docs/maintainer/adding_pkgs.html#source>`_ (where you specify where the package's source code is coming from) can be pulled from GitHub, PyPI, or other sources and sha256 hashes are always used, so moving of tags or uploading of new sdists can not cause automatic package rebuilds.
    Also, once packages are accepted and made into feedstocks, only the maintainers of that feedstock have the right to merge PRs made to that feedstock.
-2. Each feedstock can only upload packages for that feedstock. This is enforced by using a cf-staging channel where builds are first sent. 
+2. Each feedstock can only upload packages for that feedstock. This is enforced by using a cf-staging channel where builds are first sent.
    A bot then assesses that the submitting feedstock has permission to build the package it has submitted, and only then will it relay the build to the conda-forge channel.
    This helps mitigate against a bad actor gaining access to an inconspicuous feedstock and then trying to push a build with malicious code into essential infrastructure packages (e.g., OpenSSL or Python).
 3. We have `artifact-validation <https://github.com/conda-forge/artifact-validation>`__ for validating all the ``conda-forge`` artifacts before they are uploaded to ``anaconda.org``. This validation looks for various security-related items, such as artifacts that overwrite key pieces of certain packages.
 4. We have a dedicated `Security and Systems Sub-Team <https://conda-forge.org/docs/orga/subteams.html?highlight=security+team#security-and-systems-sub-team>`__ who works hard towards making sure to secure and maintain appropriate access to the credentials and services/systems used by ``conda-forge``.
+
+Significant Changes To Upstream Projects
+========================================
+
+From time to time, we make changes in upstream projects so that they better integrate into the ``conda-forge`` ecosystem. We
+have listed some, but not all, of those changes here for specific projects along with any associated documentation.
+
+Python
+------
+
+* We disable searching of registry paths on Windows in order to enable better environment isolation.
+  See this `GitHub issue <https://github.com/ContinuumIO/anaconda-issues/issues/10236>`_ and the associated
+  `patch <https://github.com/conda-forge/python-feedstock/blob/main/recipe/patches/0009-Disable-registry-lookup-unless-CONDA_PY_ALLOW_REG_PA.patch>`_ in the python feedstock.
```

Comment:
```suggestion
  `patch <https://github.com/conda-forge/python-feedstock/blob/main/recipe/patches/0009-Disable-registry-lookup-unless-CONDA_PY_ALLOW_REG_PA.patch>`_ in the python feedstock.
```

This link might break in the future, which may be a good thing because then these docs would need to be updated. Just checking that's intended instead of linking to a specific commit hash.

Reply:
I just linked to the file where I thought it was the most logical.